tests/vcr_python_wrapper.py

```diff
@@ -48,8 +48,8 @@ def query_matcher_ignore_proxy(r1, r2):
 
 
 def katello_manifest_body_matcher(r1, r2):
-    if r1.headers.get('content-type').startswith('multipart/form-data') and r2.headers.get('content-type').startswith('multipart/form-data'):
-        if r1.path.endswith('/subscriptions/upload') and r2.path.endswith('/subscriptions/upload'):
+    if r1.path.endswith('/subscriptions/upload') and r2.path.endswith('/subscriptions/upload'):
+        if r1.headers.get('content-type').startswith('multipart/form-data') and r2.headers.get('content-type').startswith('multipart/form-data'):
```

Comment:
Is this a performance thing?

Reply:
no, a GET request has no content and thus no content-type.

Reply:
so I want to match only the POST to the upload endpoint